kotlin-emotion/src/main/kotlin/emotion/react/Global.ext.kt

```diff
@@ -1,10 +1,12 @@
 package emotion.react
 
 import csstype.PropertiesBuilder
+import csstype.RuleBuilder
+import csstype.Rules
 import kotlinx.js.jso
 
 inline fun GlobalProps.styles(
-    crossinline block: PropertiesBuilder.() -> Unit,
+    crossinline block: RuleBuilder<PropertiesBuilder>.() -> Unit,
```

Comment:
What about pseudos?

Reply:
pseudos should be linked with some selector, they can be used on top level